infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/project/OpenShiftWorkspaceServiceAccount.java

```diff
@@ -97,6 +100,17 @@ private void createExecRole(OpenShiftClient osClient, String name) {
     osClient.roles().inNamespace(projectName).create(execRole);
   }
 
+  private void createViewRole(OpenShiftClient osClient, String name) {
+    Role viewRole =
+        new RoleBuilder()
+            .withNewMetadata()
+            .withName(name)
+            .endMetadata()
+            .withRules(new PolicyRuleBuilder().withResources("pods").withVerbs("list").build())
```

Comment:
Is it view role, or just view pods?

Reply:
View role allows to list pods, and this is what exec server needs?

Reply:
It makes sense to leave it named as `view` if later we will add more rules there. And even in this case `view` role would not be `view`. What do you think about workspace specific name, like `che-view`, `workspace-view`, ...

Reply:
I like workspace-view.

I has just occurred to me I need to add this role to deployment yamls too.